### ISSUE TITLE
team_results.py `FutureWarning`

### ISSUE THREAD

**Issue by notBillJames** [jldbc/pybaseball#242]
On lines 72, and 85 no keywords are passed for arguments into the `DataFrame.drop() function`. The following `FutureWarning` is returned:

```
pybaseball/pybaseball/team_results.py:72: FutureWarning: In a future version of pandas all arguments of DataFrame.drop except for the argument 'labels' will be keyword-only
  df = df.drop('',1) #not a useful column
```

It would be good to change this to avoid deprecation.

**Comment by tjburch** [comment]
Thanks for noting this, just made a quick PR to resolve.

**Comment by schorrm** [comment]
Closed by #243
